Fix: put subscription on hold when Stripe Radar blocks a renewal payment (#5170)
woocommerce/woocommerce-gateway-stripe: changelog.txt

* Fix - Improve performance of CSS style lookups
* Add - Initial implementation of always-expanded Optimized Checkout Suite in shortcode checkout
* Dev - Collapse PHPUnit tests using data providers to reduce duplication and improve test isolation

> * Fix - Put subscription on hold when Stripe Radar blocks a renewal payment to prevent WC Subscriptions from scheduling further retries that would also be blocked

Reviewer: We can drop the last part to make it concise.
```suggestion
* Fix - Put subscription on hold when Stripe Radar blocks a renewal payment to prevent WC Subscriptions from scheduling further retry attempts
```
